test/test-udp-multicast-join6.c

```diff
@@ -134,6 +134,17 @@ TEST_IMPL(udp_multicast_join6) {
 
   ASSERT(r == 0);
 
+  r = uv_udp_set_source_membership(&client, "ff02::1", NULL, "2001:db8:a0b:12f0::1", UV_JOIN_GROUP);
```

Comment:
Can the rest of the test still be run if `UV_ENODEV` or `UV_EPROTONOSUPPORT` is returned? If so, maybe the assertion should just allow them so that the remainder of the test is still executed. If that can't be done, maybe a separate test file should be created.